src/app/add-item/add-item.component.html

```diff
@@ -0,0 +1,28 @@
+<div class="container">
+    <h2 class="dialogTitle" mat-dialog-title>{{ title }}</h2>
+    <mat-dialog-content [formGroup]="dialogForm" (ngSubmit)="createActionItem(dialogForm)">
```

Comment:
Is there any reason to add this `ngSubmit` here? What's more, you are invoking `createActionItem` function with one argument but it does not accept any parameters.